_rules/inline_link_in_paragraph_is_distinguishable_be4d0c.md

```diff
@@ -0,0 +1,240 @@
+---
+id: be4d0c
+name: Inline link in paragraph is distinguishable
+rule_type: atomic
+
+description: |
+  This rule checks that links that are embedded in a paragraph have a way to distinguish them as links compared to the rest of the text
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+acknowledgements:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to any HTML element which:
+
+- has the [semantic role][] of ['link'][link] or a [semantic role][] that inherits from the ['link'][link] role; and
+- is a [descendant][] of a `p` element; and
+- has [descendant][] [text nodes][text node] that are [visible][]; and
+- the `p` element has other [descendant][] [visible][] [text nodes][text node].
+
+## Expectation
+
+Each target element has either:
+
+- An icon immediately before, after or in the link denoting it as a link
+- Text immediately before, after or in the link denoting it as a link
+- a [distinguishing style][] compared to the other text not based on color alone
+- a different color compared to the rest of the text in the paragraph that has at least a 3:1 [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) difference with the rest of the text and a [distinguishing style][] both when the target element [gains focus][gain focus] and the target element is [hovered][]
+
+## Assumptions
+
+This rule assumes that the link is distinguishable from the rest of the text with color, which means it fails SC 1.4.1 when there is not another way to distinguish it.
```

Comment:
I think these assumptions are relevant to this rule as well:
https://github.com/act-rules/act-rules.github.io/blob/develop/__deprecated__/_drafts/outdated/SC1-4-1-link-style-in-text.md#assumptions